mypy/plugin.py

```diff
@@ -303,20 +303,16 @@ def builtin_type(self, fully_qualified_name: str) -> Instance:
         raise NotImplementedError
 
     @abstractmethod
-    def lookup_fully_qualified(self, name: str) -> SymbolTableNode:
+    def lookup_fully_qualified(self, name: str) -> Optional[SymbolTableNode]:
         """Lookup a symbol by its fully qualified name.
 
-        Raise an error if not found.
+        Return None if not found.
         """
         raise NotImplementedError
 
-    @abstractmethod
     def lookup_fully_qualified_or_none(self, name: str) -> Optional[SymbolTableNode]:
-        """Lookup a symbol by its fully qualified name.
```

Comment:
@hauntsaninja I'm not sure whether it'a good way to keep an un-abstract method in an Interface.